src/stage1/exe_copy.rs

```diff
@@ -50,7 +54,14 @@ impl ExeCopy {
 
     fn get_libs_for(&mut self) -> Result<()> {
         trace!("get_libs_for: entered");
-        let ldd_path = whereis("ldd").upstream_with_context("Failed to locate ldd executable")?;
+        let ldd_path: String;
+        if self.ldd_script_path.is_empty() {
+            debug!("lld path was not provided, will use the one from current OS, if available");
+            ldd_path = whereis("ldd").upstream_with_context("Failed to locate ldd executable, please provide path to ldd manually")?;
+        } else {
+            debug!("Provided ldd path: {}", self.ldd_script_path.display());
+            ldd_path = self.ldd_script_path.as_path().display().to_string();
+        }
```

Comment:
Like in a case I commented earlier, I think it would make sense to keep the inclusion and the usage of the ldd-path option on a single commit.

Reply:
Sure, added in the same commit in https://github.com/balena-os/takeover/pull/66/commits/4bbfe8aa7849ff33f88ee6951fe93c4999633533